src/commonMain/kotlin/com/charleskorn/kaml/YamlConfiguration.kt

```diff
@@ -52,6 +54,8 @@ public data class YamlConfiguration(
     internal val allowAnchorsAndAliases: Boolean = false,
     internal val yamlNamingStrategy: YamlNamingStrategy? = null,
     internal val codePointLimit: Int? = null,
+    @ExperimentalSerializationApi
```

Comment:
I don't think there's a need to mark this as experimental.

Reply:
All the new api features are marked as experimental in the `kotlinx.serialization` library when they are introduced. As a matter of fact, its their convention.